partner_contact_birthdate/__manifest__.py

```diff
@@ -1,5 +1,6 @@
 # Copyright 2014-2015  Grupo ESOC <www.grupoesoc.es>
 # Copyright 2017-Apertoso N.V. (<http://www.apertoso.be>)
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
```

Comment:
Leave out the `(C)` and replace `Today:` with 2020

Reply:
Same goes for all other occurences of this line in the PR.

Reply:
@NL66278  done